app/src/repositories/ReservationRepository.ts

```diff
@@ -222,6 +222,21 @@ const ReservationRepository: CommonRepositoryInterface<Reservation> & ShopServic
     return cleanReservations
   },
 
+  async fetchShopReservationsBySS(shopId, stylistId?) {
```

Comment:
we should also add the date of the reservation so it won't get unnecessary reservations